packages/router/src/router_module.ts

```diff
@@ -272,24 +267,15 @@ export interface ExtraOptions {
   useHash?: boolean;
 
   /**
-   * One of `enabled` or `disabled`.
-   * When set to `enabled`, the initial navigation starts before the root component is created.
-   * The bootstrap is blocked until the initial navigation is complete. This value is required for
-   * [server-side rendering](guide/universal) to work.
-   * When set to `disabled`, the initial navigation is not performed.
-   * The location listener is set up before the root component gets created.
-   * Use if there is a reason to have more control over when the router
+   * One of `enabled`, `enabledBlocking`, `enabledNonBlocking` or `disabled`.
+   * When set to `enabled` or `enabledBlocking`, the initial navigation starts before the root
+   * component is created. The bootstrap is blocked until the initial navigation is complete. This
+   * value is required for [server-side rendering](guide/universal) to work. When set to
+   * `enabledNonBlocking`, the initial navigation starts after the root component has been created.
+   * The bootstrap is not blocked until the initial navigation is complete. When set to `disabled`,
```

Comment:
I think this could be a bit less ambiguous...

```suggestion
   * The bootstrap is not blocked waiting for the initial navigation is complete. When set to `disabled`,
```

Reply:
Or

```suggestion
   * The bootstrap is not blocked on the completion of the initial navigation. When set to `disabled`,
```